tests/src/Kernel/GroupManagerSubscriptionTest.php

```diff
@@ -117,6 +117,26 @@ public function testGroupManagerSubscription($group_has_owner, $membership_is_ov
     $this->assertEquals($membership_is_overridden, $this->isMembershipOverridden($membership));
   }
 
+  /**
+   * Tests 'Automatically add creators to the group' admin settings.
+   *
+   * @dataProvider autoAddOwnersSettingProvider
+   */
+  public function testAutoAddOwnersSetting($auto_add_group_owner_membership) {
+    $config = $this->container->get('config.factory')->getEditable('og.settings');
+    $config->set('auto_add_group_owner_membership', $auto_add_group_owner_membership);
+    $config->save();
+    $group = Node::create([
+      'title' => 'membership is not overridden',
+      'type' => 'group',
+      'uid' => $this->owner->id(),
+    ]);
+    $group->save();
+    $membership = $this->membershipManager->getMembership($group, $this->owner->id());
+    // Membership should be created only when auto_add_group_owner_membership is enabled.
```

Comment:
Line exceeds 80 characters; contains 89 characters 

```suggestion
    // Membership should be created only when auto_add_group_owner_membership is
    // enabled.
```